tools/system_libs.py

```diff
@@ -1127,7 +1127,8 @@ def get_files(self):
       'stdlib_exception.cpp',
       'stdlib_stdexcept.cpp',
       'stdlib_typeinfo.cpp',
-      'private_typeinfo.cpp'
+      'private_typeinfo.cpp',
+      'format_exception.cpp'
```

Comment:
Can you add a trailing comma here?  (I know it was missing before, but it helps make modifications to the list easier in the future)